docker-compose.yml

```diff
@@ -1,15 +1,15 @@
 version: "2.1"
 services:
   db:
-    image: mysql:8.0.28-oracle
+    image: edxops/mysql:5.7
     container_name: enterprise_access.db
     environment:
       MYSQL_ROOT_PASSWORD: ""
       MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
     networks:
       - devstack_default
     volumes:
-      - enterprise_access_mysql8:/var/lib/mysql
+      - enterprise_access_mysql57:/var/lib/mysql
```

Comment:
[inform] edX Ops team now publishes a cross-compatible mysql 5.7 image (for sake of M1 mac devs).